test/cli/cli_test.hpp

```diff
@@ -93,3 +93,6 @@ struct cli_test : public ::testing::Test
         }
     }
 };
+
+// To prevent issues when running multiple CLI tests in parallel, give each CLI test unique names:
+struct fastq_to_fasta : cli_test {};
```

Comment:
This is inheritance – isn't there an access specifier needed?
```suggestion
struct fastq_to_fasta : public cli_test {};
```

Reply:
`cli_test` is a struct, so the default visibility is `public`, but I agree that it is better to be verbose